src/App.tsx

```diff
@@ -1,9 +1,55 @@
 import './App.scss';
+import {
+  NavLink,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { useState } from 'react';
+import { HomePage } from './components/HomePage';
+import { PeopleTable } from './components/PeopleTable';
+import { PageNotFound } from './components/PageNotFound';
+import { getPeople } from './api';
+import { Person } from './Types/Person';
+import 'bulma/css/bulma.css';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+export const App = () => {
+  const [people, setPeople] = useState<Person[]>([]);
 
-export default App;
+  getPeople()
+    .then(peopleFromServer => setPeople(peopleFromServer));
+
+  const preparedPeople = people
+    .map(currentPerson => ({
+      ...currentPerson,
+      mother: people.find(person => person.name === currentPerson.motherName),
+      father: people.find(person => person.name === currentPerson.fatherName),
+    }));
+
```

Comment:
Fetch people from [API](https://mate-academy.github.io/react_people-table/api/people.json) when PeoplePage is opened